test/e2e/action_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
```

Comment:
for curiosity, why do we need this line, and what it does do? Is it something different than `+build integration`

Reply:
Because in 1.17 they changed the syntax for build tags. The `go fmt` adds both of them to gives you an option to easily migrate your code.

I recommend you to take a look at:
- https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md
- or recording https://www.youtube.com/watch?v=AgR_mdC4Rs4
in a free time, to learn why they changed it.

Reply:
Thanks, I will check :+1: